t/unittest/access_filter/radius.t

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+radius
+
+=head1 DESCRIPTION
+
+unit test for radius
+
+=cut
+
+use strict;
+use warnings;
+#
+use lib '/usr/local/pf/lib';
+
+our @tests;
+
+BEGIN {
+    #include test libs
+    use lib qw(/usr/local/pf/t);
+
+    #Module for overriding configuration paths
+    use setup_test_config;
+    @tests = (
+        {
+            name           => 'reply:Packetfence-Raw',
```

Comment:
I believe this attribute supports array, can you test with an array too

Reply:
These tests now invalid

Reply:
Done